src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/ISnapshotResolver.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+internal interface ISnapshotResolver
+{
+    /// <summary>
+    /// Finds all the projects with a directory that contains the document path. 
```

Comment:
This makes it sound like if I include a document from `C:\Foo` in a project that is in `C:\Bar` it will be found. Suggest rewording to something like:

```suggestion
    /// Finds all the projects where the document path starts with the path of the folder that contains the project file.
```